hub/api/dataset.py

```diff
@@ -344,6 +344,36 @@ def delete(self):
             self.client.delete_dataset_entry(self.org_id, self.ds_name)
             logger.info(f"Hub Dataset {self.path} successfully deleted.")
 
+    @staticmethod
+    def delete_at(path: str, force: bool = False, large_ok=False):
+        """Deletes the dataset at a given path.
+        This is an IRREVERSIBLE operation. Data once deleted can not be recovered.
+
+        Args:
+            path (str): The path to the dataset to be deleted.
+            force (bool): Delete data regardless of whether
+                it looks like a hub dataset. All data at the path will be removed.
+            large_ok (bool): Delete datasets larger than 1GB. Disabled by default.
+        """
+        base_storage = get_storage_provider(
+            path, storage=None, read_only=False, creds={}, token=None
+        )
+
+        if force:
+            base_storage.clear()
+        else:
+            ds = Dataset(path)
+            if not large_ok:
+                tensors = ds.tensors.values()
+                chunk_engines = [tensor.chunk_engine for tensor in tensors]
+                size = sum(c.num_chunks * c.min_chunk_size for c in chunk_engines)
```

Comment:
this looks like a new method. can you create one called `ds.approximate_size()`?